### PR TITLE
Add issue and PR templates

### PR DESCRIPTION
**Pull request by sarahsporck**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/technical-task.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: Technical Task \U0001F4A1
```
Comment (SteffiStuffel) [inline .github/ISSUE_TEMPLATE/technical-task.md:2]: I think the name "technical Task" is a bit confusing, as this is a feature request isn't it? Or are we missing the feature request template?

Comment (sarahsporck) [reply .github/ISSUE_TEMPLATE/technical-task.md:2]: Feature === Technical Task. I copied this from the entitlementcard template tbh. I'm also fine with feature. Should we also rename the label (Task -> Feature) then?

Comment (SteffiStuffel) [reply .github/ISSUE_TEMPLATE/technical-task.md:2]: Oke, for the technical task sounds like a version update or a refactoring, but I am fine with this, if this is the correct term for a feature

Comment (LeandraH) [reply .github/ISSUE_TEMPLATE/technical-task.md:2]: I guess the feature request is then handled by the user story, and the technical task is actually more of a version update thing?

**Review by LeandraH — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/technical-task.md (hunk 1 above)
Comment (steffenkleinle) [reply .github/ISSUE_TEMPLATE/technical-task.md:2]: I also think the term `Technical Task` is a little misleading. What is the difference between a `User Story` and a `Technical Task` then if a `Technical Task` is a feature? I have the same definition in my head as @SteffiStuffel that a technical task is more refactoring/version updates/... that doesn't add a new "feature" or behavior

Comment (SteffiStuffel) [reply .github/ISSUE_TEMPLATE/technical-task.md:2]: Difference between User Story and Feature is normally that a User Story is bigger than a Feature. You need multiple features to achieve a User Story and multiple User Stories to achieve an epic. 
A User Story is also not necessarily in a good size for a development ticket.

**Review by steffenkleinle — APPROVED** [review]

**Comment by sarahsporck** [comment]
We decided to go for the term `task` instead of `technical task`.

**Pull request merged**